### PR TITLE
safeguard for filesystem repo in db seed

### PR DESCRIPTION
**Pull request by ulferts**
Add a safeguard around repository creation in db:seed.

The repo will only be created if filesystem repos are properly configured.

Problem arose when fixing https://community.openproject.org/work_packages/19150

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://ulferts-hotfix-filesystem-repo-in-dev-seed.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://ulferts-hotfix-filesystem-repo-in-dev-seed.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: db/seeds/development.rb [hunk 1]
```diff
@@ -107,11 +107,30 @@
   time_entry_activities << time_entry_activity
 end
 
-Setting.enabled_scm = Setting.enabled_scm.dup << 'Filesystem' unless Setting.enabled_scm.include?('Filesystem')
+repo_url_setting = OpenProject::Configuration['scm_filesystem_path_whitelist']
+repository = if repo_url_setting.empty?
+               puts <<-MESSAGE
```
Comment (ulferts) [inline db/seeds/development.rb:112]: @houndci, what do you know. This is a shell script.

Comment (NobodysNightmare) [inline db/seeds/development.rb:112]: A shell-script ending in `.rb` you mean ^^

Comment (ulferts) [inline db/seeds/development.rb:112]: I meant a script executed in the shell.

**Comment by NobodysNightmare** [comment]
LGTM :+1:

**Pull request merged**